lib/views/index.erb

```diff
@@ -0,0 +1 @@
+<p> <%=@name%> is a student at craft academy</p>
```

Comment:
Embedded ruby should have spaces before and after code. It's a readability issue.
```
<%= @name %>
```